lbrynet/lbryfilemanager/EncryptedFileManager.py

```diff
@@ -74,12 +75,49 @@ def _add_to_sd_identifier(self):
         self.sd_identifier.add_stream_downloader_factory(
             EncryptedFileStreamType, downloader_factory)
 
-    def _start_lbry_files(self):
+    def _check_stream_is_managed(self, stream_hash):
+        # check that all the streams in the stream_info_manager are also
+        # tracked by lbry_file_manager and fix any streams that aren't.
+        def _check_rowid_result(rowid):
+            if rowid is not None:
+                return defer.succeed(True)
+            raise MissingLBRYFile
```

Comment:
this should be `raise MissingLBRYFile()` - ideally though with some info about what is missing